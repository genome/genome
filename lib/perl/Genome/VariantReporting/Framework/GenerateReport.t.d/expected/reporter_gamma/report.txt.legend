Headers
exp1	Expert 1 info field
Filters
filter_one	FIRST_FILTER	The first filter
filter_two	OTHER_FILTER	The other filter
