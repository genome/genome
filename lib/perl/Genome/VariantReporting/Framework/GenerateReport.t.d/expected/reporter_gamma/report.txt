exp1
A
C
C
-
